controller:
  lambda_gain: 11.5
  eta_gain: 0.15

impedance:
  Mass: 1.0
  Damping: 1.0
  Stiff: 0.0
  sensor_used: true